tests/facts/windows_services/windows_services.json

```diff
@@ -0,0 +1,10240 @@
+{
```

Comment:
Could we trim this test down a bit too? Wary of +20k lines!

Reply:
Sure. Found another bug!